format_spec/fragment.md

```diff
@@ -271,3 +301,49 @@ The on-disk format of each data file is:
 | … | … | … |
 | Tile N | [Tile](./tile.md#tile) | The data of tile N |
 
+## Legacy coordinates file
+
+Prior to version 5, dimension data for sparse cells are combined in a single tile that is stored in the `__coords.tdb` file. The tile is filtered with the filters specified in the _Coords filters_ field of the [array schema](./array_schema.md).
+
+Coordinates of a multi-dimensional array are placed in either zipped or unzipped order. In zipped order, coordinates of a cell are placed next to each other and ordered by the cell index, while in unzipped order, all coordinates values of a dimension are placed next to each other and ordered by the dimension index.
+
+* Since version 2, coordinates are always stored unzipped.
+* In version 1, coordinates are stored unzipped if a [compression filter](./tile.md#compression-filters) exists in the filter list, otherwise they are stored zipped.
```

Comment:
```suggestion
* In version 1, coordinates are stored unzipped if a [compression filter](./tile.md#compression-filters) exists in the filter list. Otherwise, they are stored zipped.
```

Reply:
Done.